docs/src/02-metadata-merging.jmd

```diff
@@ -186,10 +191,13 @@ filter!(r-> !any(ismissing, [r[:studyID], r[:sampleID], r[:timepoint], r[:metada
 disallowmissing!(allmeta)
 
 # show a random assortment of ~ 10 rows
+# **shows random assortment of 4 rows x 6 columns**
+# **What does it mean when `sampleID`s end in `M`. Not clear what that's meant to mean.**
+# **May also be worthwhile to explain naming convention of fecal `sampleID`s**
```

Comment:
Do you want to add a paragraph in somewhere it makes sense? I have it [documented here](https://github.com/Klepac-Ceraj-Lab/echo_analysis/blob/master/src/metadata_handling.jl#L1-L13), and I originally was going to have the docstrings reported out (I'd still like to fix that in the long run), but I agree it would be good to put it up front somewhere.